src/cljs/main/broadfcui/common/table.cljs

```diff
@@ -118,7 +119,7 @@
          (when (and (:filterable? behavior) (not (contains? external-query-params :filter-text)))
            (let [filter-bar-props (:filter-bar toolbar)]
              [:div {:style (:style filter-bar-props)}
-              [comps/TextFilter (merge
+              [text-filter/TextFilter (merge
```

Comment:
Since this is the only thing in this file, you can `:refer [TextFilter]` in the imports and just use `[TextFilter ...` here.